demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -0,0 +1,255 @@
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        validator_results: List[ValidationResult] = []
+        for content_item in content_items:
+            if content_item.pack_metadata_dict and content_item.pack_metadata_dict.get(
+                "hidden"
+            ):
+                continue
+            (
+                invalid_headers_content_type,
+                invalid_headers_content_item,
+            ) = self.validate_release_notes_headers(content_item)
+            content_type_message = (
+                "Headers Content Types: {}\n".format(
+                    ", ".join(invalid_headers_content_type)
+                )
+                if invalid_headers_content_type
+                else ""
+            )
+            content_item_message = (
+                "Headers Content Items: {}\n".format(
+                    ", ".join(invalid_headers_content_item)
+                )
+                if invalid_headers_content_item
+                else ""
+            )
+            if invalid_headers_content_type or invalid_headers_content_item:
+                validator_results.append(
+                    ValidationResult(
+                        validator=self,
+                        message=self.error_message.format(
+                            content_type_message=content_type_message,
+                            content_item_message=content_item_message,
+                        ),
+                        content_object=content_item,
+                    )
+                )
+        return validator_results
+
+    def remove_none_values(self, ls: Union[List[Any], Tuple[Any, ...]]) -> List[Any]:
+        """
+        Filters out None values from a list or tuple.
+
+        Args:
+            ls (List or Tuple): The list or tuple to filter.
+
+        Returns:
+            List: Filtered list with None values removed.
+        """
+        return list(filter(None, ls))
+
+    def extract_rn_headers(self, release_note_content: str) -> Dict[str, List[str]]:
+        """
+        Extracts the headers from the release notes file.
+
+        Args:
+            release_note_content (str): Content of the release notes file.
+
+        Return:
+            Dict[str, List[str]]: A dictionary representation of the release notes file that maps
+                                  content types' headers to their corresponding content items' headers.
+        """
+        headers: Dict[str, List[str]] = {}
+        content_type_section_pattern = re.compile(CONTENT_TYPE_SECTION_REGEX, re.M)
+        content_item_section_pattern = re.compile(CONTENT_ITEM_SECTION_REGEX, re.M)
+
+        # Get all sections from the release notes using regex
+        rn_sections = content_type_section_pattern.findall(release_note_content)
+        for section in rn_sections:
+            section = self.remove_none_values(ls=section)
+            if not section:
+                continue
```

Comment:
add a debug log saying that we ignore an empty section

Reply:
added